infra/lib/infra/event.rb

```diff
@@ -1,3 +1,4 @@
+require "active_support/isolated_execution_state"
 require "active_support/core_ext/hash"
```

Comment:
Any reason to include this? It has been already reverted once so I'd like to see the backtrace why it reappears.